lib/forcegraph/draw.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+define(['helper'], function (helper) {
+  var self = this;
+
+  var ctx;
+  var transform;
+
+  var highlight;
+
+  const nonUplinkColor = '#F2E3C6';
+  const locationColor = '#00ff00';
+  const noLocationColor = '#0000ff';
+  const clientColor = 'rgba(230, 50, 75, 1.0)';
+
+  const cableColor = '#50B0F0';
```

Comment:
Jep, also for https://github.com/ffrgb/meshviewer/blob/develop/lib/map/clientlayer.js#L46 -> later